src/systemctl/systemctl-show.c

```diff
@@ -1834,6 +1835,31 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                                 return bus_log_parse_error(r);
 
                         return 1;
+                } else if (contents[0] == SD_BUS_TYPE_STRUCT_BEGIN && streq(name, "OpenFile")) {
+                        char *path, *fdname;
+                        uint64_t offlags;
+
+                        r = sd_bus_message_enter_container(m, SD_BUS_TYPE_ARRAY, "(sst)");
+                        if (r < 0)
+                                return bus_log_parse_error(r);
+
+                        while ((r = sd_bus_message_read(m, "(sst)", &path, &fdname, &offlags)) > 0) {
+                                bus_print_property_valuef(name, expected_value, flags, "%s", open_file_to_string(&(OpenFile){
```

Comment:
This call to `open_file_to_string()` will leak memory. You should define a local variable with `_cleanup_free` and assign the result of `open_file_to_string()` to that variable to make sure the memory gets cleanup up.

Also, no need for `bus_print_property_valuef()` here, you can just use `bus_print_property_value()`